src/server/json_family.cc

```diff
@@ -591,7 +591,7 @@ OpResult<long> OpDel(const OpArgs& op_args, string_view key, string_view path) {
   long total_deletions = 0;
   if (path.empty()) {
     auto& db_slice = op_args.shard->db_slice();
-    auto [it, _] = db_slice.FindExt(op_args.db_cntx, key);
+    auto it = db_slice.FindMutable(op_args.db_cntx, key).it;  // We can ignore post_updater
```

Comment:
why we can ignore post upater? what does it mean ignore?

Reply:
By ignore I meant that we can let `PostUpdate()` run immediately. I'm open to rephrasing however you think will be clear.